changelog-template.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "00000000-0000-0000-0000-000000000000",
+    "type": "feature|bugfix|dependency",
+    "description": "Description of your changes",
```

Comment:
do we need collapse here

Reply:
The doc mentions it, don't think we need it here, but also a default of false would be fine.